src/test/java/dev/group1/RevatureConnectPlus/TestUserCount.java

```diff
@@ -0,0 +1,40 @@
+package dev.group1.RevatureConnectPlus;
+
+public class TestUserCount{
```

Comment:
I'm a little confused what this class does. It's in test so I'm not worried about it, but I'd love to ask you about it tomorrow